packages/server/src/sdk/app/views/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { ViewV2 } from "@budibase/types"
+import { utils, dataFilters } from "@budibase/shared-core"
+import { isPlainObject } from "lodash"
+import { HTTPError } from "@budibase/backend-core"
+
+function isEmptyObject(obj: any) {
+  return obj && isPlainObject(obj) && Object.keys(obj).length === 0
```

Comment:
Are these checks ( `isPlainObject(obj)` && `Object.keys(obj).length === 0`) not redundant?

Reply:
Not quite, the problem is arrays.

```typescript
> _.isPlainObject({})
true
> _.isPlainObject([])
false
> Object.keys({}).length === 0
true
> Object.keys([]).length === 0
true
```